blockchain/types/tx_internal_data.go

```diff
@@ -373,6 +383,8 @@ type TxInternalDataPayload interface {
 type TxInternalDataEthTyped interface {
 	setSignatureValues(chainID, v, r, s *big.Int)
 	GetAccessList() AccessList
+	GetGasTipCap() *big.Int
+	GetGasFeeCap() *big.Int
```

Comment:
How about separate the interface for TxInternalDataDynamicFee?
Those functions are used only for dynamicFeeTx, so AccessListTx has to implement this feels unnecessary.